test/integration/install_test.go

```diff
@@ -258,9 +270,10 @@ func TestInstallOrUpgradeCli(t *testing.T) {
 	var (
 		cmd  = "install"
 		args = []string{
+			// Use --set flags once these releases into edge
```

Comment:
OKay, Now the tests are added to check if the secret is same even after upgrades.

Trying to think out loud, Currently, these flags are only available on the edge and hence this has to be gated to edges but still these overrides are not stored into the secret in the edge, and hence failing this test. So, We have to wait for a edge release with this change, and then using the `--set` flags here. WDYT? @alpeb